src/js/constants/ContainerConstants.js

```diff
@@ -1,22 +1,22 @@
 const ContainerConstants = {
-  NETWORK: {
+  network: Object.freeze({
```

Comment:
I'm sorry, I misunderstood the problem you were solving here. I thought `Object.freeze()` would work recursively, but – NOPE. 

Should we consider something like: https://github.com/substack/deep-freeze ?